test/integration-test.py

```diff
@@ -164,6 +164,20 @@ def wait_for_ocsp_good(cert_file, issuer_file, url):
 def wait_for_ocsp_revoked(cert_file, issuer_file, url):
     fetch_until(cert_file, issuer_file, url, ": good", ": revoked")
 
+def reset_akamai_purges():
+    requests.post("http://localhost:6789/debug/reset-purges")
+
+def verify_akamai_purge():
+    response = requests.get("http://localhost:6789/debug/get-purges")
+    purgeData = response.json()
+    if os.environ.get('BOULDER_CONFIG_DIR', '').startswith("test/config-next"):
+        if len(purgeData["V3"]) is not 1:
+            raise Exception("Unexpected number of Akamai v3 purges")
```

Comment:
Should there also be a check that when the config is config-next that there are no v2 purges (and vice-versa for config)?

Reply:
Yup, done.